test/reporter_output_test.cc

```diff
@@ -16,6 +16,7 @@ static int AddContextCases() {
   AddCases(TC_ConsoleErr,
            {
                {"%int[-/]%int[-/]%int %int:%int:%int$", MR_Default},
+               {"System Name", MR_Next},
```

Comment:
What is "system name"?
How is this different from "host name"?
If it is exactly the "host name", just use that term?